treeherder/perf/auto_perf_sheriffing/perf_sheriff_bot.py

```diff
@@ -37,14 +37,17 @@ def __init__(
         secretary_tool: SecretaryTool,
         notify_client: taskcluster.Notify,
         max_runtime: timedelta = None,
+        email_writer: EmailWriter = None,
     ):
         self.report_maintainer = report_maintainer
         self.backfill_tool = backfill_tool
         self.secretary = secretary_tool
         self._notify = notify_client
+        self._max_runtime = self.DEFAULT_MAX_RUNTIME if max_runtime is None else max_runtime
```

Comment:
Can't this line be written as well like `self._max_runtime = max_runtime or self.DEFAULT_MAX_RUNTIME` ?

Reply:
Oh... Definitely! Thanks for the catch! 🎣